static/app/views/settings/organizationMembers/components/inviteMembersButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import FeatureDisabled from 'sentry/components/acl/featureDisabled';
+import Button from 'sentry/components/button';
+import {Hovercard} from 'sentry/components/hovercard';
+import {IconMail} from 'sentry/icons';
+import {t} from 'sentry/locale';
+
+type InviteMembersButtonProps = {
+  onClick: () => void;
+  disabled?: boolean;
+};
+
+function InviteMembersButton({disabled, onClick}: InviteMembersButtonProps) {
+  const action = (
+    <Button
+      priority="primary"
+      size="small"
+      onClick={onClick}
+      data-test-id="email-invite"
```

Comment:
Does this get used?

Reply:
the test id was there before, so i left it the way it was. 

the tests that i saw are calling `wrapper.find('StyledButton');` which seems not as specific as they could be